docs/architecture.md

```diff
@@ -17,29 +17,28 @@ A **Trace** is a data/execution path through the system, and can be thought of a
 ![Architecture](images/architecture.png)
 *The current Jaeger architecture*
 
-This section details the constituents of Jaeger, and how they relate to each other. It is arranged by the order in which spans from your application interact with them. 
+This section details the constituents of Jaeger and how they relate to each other. It is arranged by the order in which spans from your application interact with them. 
 
-###Jaeger client
-Jaeger clients are language specific implementations of the OpenTracing API. The contain a Tracer, which is an interface for Span creation, and context propagation. 
+###Jaeger client libraries
+Jaeger clients are language specific implementations of the OpenTracing API. They contain a Tracer, which is an interface for Span creation, and a mechanism for context propagation. 
```

Comment:
and a mechanism for context propagation is a lie.

How about, "a mechanism to extract tracing specific metadata from propagated context"?